.github/workflows/policy-check.yaml

```diff
@@ -24,4 +24,4 @@ jobs:
         # Every file in the repo that is treated as text
         FILES2CHECK=$(git ls-files | xargs git grep --name-only -I -e . --)
 
-        python3.8 -W ignore ./scripts/policycheck.py 0.5 ./.github/workflows/policy-ignored-lines.txt $FILES2CHECK
+        python3.8 -W ignore ./tools/policycheck.py 0.5 ./.github/workflows/policy-ignored-lines.txt $FILES2CHECK
```

Comment:
Do we want to move this to a new `.github/actions`? See https://help.github.com/en/actions/building-actions/about-actions#choosing-a-location-for-your-action

Reply:
Yes, in my next PR which converts this into an action and adds more custom style actions :D